client/src/extended_bootstrap_css/bootstrap-wet-fixes-extensions.scss

```diff
@@ -268,25 +268,32 @@ div.label {
   background-color: $primaryColor !important;
   color: $textLightColor !important;
 }
-
-/* to be used with light-background */
-a.btn-ib-light {
-  color: inherit;
+.btn-ib-opened {
+  background-color: $backgroundColor;
+  border: 2px solid $backgroundColor;
+  color: $primaryColor !important;
+  border-radius: $standardBorderRadius;
+  margin: 0px 5px 0px 0px;
 }
```

Comment:
Maybe this should be named `.btn-ib-light--reversed` or something? It's specific to the light buttons, and, outside of the context of the buttons for those dropdowns, a button being "opened" doesn't mean much. Should also give it its own, `:hover` styles, etc.

Reply:
agreed with naming, but for hover styles, I intentionally left them blank to give it a more of "opened" look, so that colour stays "opened" all the time. Just tried it out with hover styles applied and it looks a bit off